### PR TITLE
Update documentation of property population.

### PR DESCRIPTION
**Pull request by ralcon**
(as described in https://github.com/prescottprue/react-redux-firebase/issues/372#issuecomment-356801514)

### Relevant Issues
#372

### CODE REVIEW

**Comment by prescottprue** [comment]
Not sure I understand what is being changed here or its effect?

Looked at the comment, and it seems I just pointed out how to use multiple populates. Then when looking at the diff of the commit I didn't see any multiple populates, just renaming of variables in a way that shouldn't change anything.

File: docs/populate.md [hunk 1]
```diff
@@ -10,18 +10,18 @@ import { compose } from 'redux'
...
-const populates = [
+const populatesOwnerWithUser = [
   { child: 'owner', root: 'users' } // replace owner with user object
 ]
 
 const enhance = compose(
   firebaseConnect([
     // passing populates parameter also creates all necessary child queries
-    { path: 'todos', populates }
+    { path: 'todos', populates: populatesOwnerWithUser }
```
Comment (prescottprue) [inline docs/populate.md]: What does this renaming do here? Is it intended to more clearly indicate the name `populates`?

**Comment by ralcon** [comment]
My fault. Something like JS newbe. Now understanding passing properties.
Passing `{ populates }` is the same like `{ populates: populates }`.

**Comment by prescottprue** [comment]
@ralcon Glad you reached out anyway! The examples/docs use some short hand like that to keep things clean, but it can make some other things easy to miss (hard balance to strike).

**Pull request closed without merging**